src/applications/ezr/tests/unit/utils/helpers/submit-transformer.unit.spec.jsx

```diff
@@ -4,113 +4,278 @@ import formConfig from '../../../../config/form';
-        },
-      };
-      const expectedResult = JSON.stringify({
-        asyncCompatible: true,
-        form: JSON.stringify({
-          veteranFullName: {
-            first: 'Jane',
-            last: 'Doe',
-          },
-          veteranSocialSecurityNumber: '234243444',
-          medicareClaimNumber: '7AD5WC9MW60',
-          medicarePartAEffectiveDate: '2009-01-02',
-          isMedicaidEligible: true,
-          isEnrolledMedicarePartA: true,
-          deductibleMedicalExpenses: 234,
-          deductibleFuneralExpenses: 11,
-          deductibleEducationExpenses: 0,
-          veteranGrossIncome: 234234,
-          veteranNetIncome: 234234,
-          veteranOtherIncome: 0,
-          veteranAddress: {
-            country: 'USA',
-            street: '123 elm st',
-            city: 'Northampton',
-            state: 'MA',
-            postalCode: '01060',
-          },
-          maritalStatus: 'never married',
-          privacyAgreementAccepted: true,
-          veteranDateOfBirth: '1990-01-01',
-          gender: 'F',
-          veteranHomeAddress: {
-            country: 'USA',
-            street: '123 elm st',
-            city: 'Northampton',
-            state: 'MA',
-            postalCode: '01060',
-          },
-          dependents: [],
-        }),
-      });
-      expect(submitTransformer(formConfig, form)).to.deep.equal(expectedResult);
-    });
+            data: {
+              veteranFullName: {
+                first: 'Jane',
+                last: 'Doe',
+              },
+              veteranSocialSecurityNumber: '234243444',
+              veteranDateOfBirth: '1990-01-01',
+              gender: 'F',
+              medicareClaimNumber: '7AD5WC9MW60',
+              medicarePartAEffectiveDate: '2009-01-02',
+              'view:isMedicaidEligible': {
+                isMedicaidEligible: true,
+              },
+              'view:isEnrolledMedicarePartA': {
+                isEnrolledMedicarePartA: true,
+              },
+              'view:deductibleMedicalExpenses': {
+                deductibleMedicalExpenses: 234,
+              },
+              'view:deductibleFuneralExpenses': {
+                deductibleFuneralExpenses: 11,
+              },
+              'view:deductibleEducationExpenses': {
+                deductibleEducationExpenses: 0,
+              },
+              'view:veteranGrossIncome': {
+                veteranGrossIncome: 234234,
+              },
+              'view:veteranNetIncome': {
+                veteranNetIncome: 234234,
+              },
+              'view:veteranOtherIncome': {
+                veteranOtherIncome: 3454,
+              },
+              'view:spouseGrossIncome': {
+                spouseGrossIncome: 75454,
+              },
+              'view:spouseNetIncome': {
+                spouseNetIncome: 2656,
+              },
+              'view:spouseOtherIncome': {
+                spouseOtherIncome: 324,
+              },
```

Comment:
Because we're using a feature toggle and the V1 pages still exist, I added in the V1 view fields to this test because they should up in the form data.